unstructured_inference/models/unstructuredmodel.py

```diff
@@ -55,6 +64,106 @@ def __call__(self, x: Image) -> List[LayoutElement]:
         """Inference using function call interface."""
         return super().__call__(x)
 
+    @staticmethod
+    def deduplicate_detected_elements(
+        elements: List[LayoutElement],
+        min_text_size: int = 15,
+    ) -> List[LayoutElement]:
+        """Deletes overlapping elements in a list of elements. Also will delete elements
+        of less than min_text_size pixels height"""
+
+        if len(elements) <= 1:
+            return elements
+
+        def clean_tables(elements: List[LayoutElement]) -> Iterable[LayoutElement]:
```

Comment:
this and the function below `enhance_regions` are important unit steps in the code logic so it would be better to refactor them out of this parent function scope so that:
- logic is isolated
- can test cleanly the unit steps instead of having to test dedupe as a whole only